vraptor-site/content/en/docs/testing-components-and-controllers.html

```diff
@@ -94,6 +94,42 @@
 
 Note that any call of the type `result.use(...)` will be ignored.
 
+##MockSerializationResult
+
+Very similar to `MockResult` with a difference that we can get the serialized content from `Result`.
```

Comment:
the same here: *"in a similar way, you can use the `MockSerializationResult` 
anytime you want to inspect the serialized content in `Result`"* (or something)